vignettes/Introduction_agestructEPICYST.Rmd

```diff
@@ -0,0 +1,164 @@
+
+```{r, fig.align='center',fig.width=7, fig.height=5}
+# Plot the output in base R
+plot(m3$Pig_Cysticercosis_prev*100 ~ m3$t, type="l", col="darkmagenta", ylim=c(0,20), ylab = 'prevalence (%)', xlab = 'time (months)')
+lines(m3$Human_Cysticercosis_prev*100 ~ m3$t, type="l", col="deeppink")
+lines(m3$Human_Taeniasis_prev*100 ~ m3$t, type="l", col="dodgerblue")
+legend('topright', c('porcine cysticercosis', 'human cysticercosis', 'human taeniasis'), lty = c(1, 1, 1),
+       col=c('darkmagenta', 'deeppink', 'dodgerblue'))
+```
+
+
+
+## Diagnostic adjustment to adjust true modelled prevalence to observed modelled prevalence
+
+The user can also specify diagnostic parameters in `set_up()` to adjust modelled prevalence to observed/apparent prevalence by providing the sensitvity and specificity characteristics of a particular diagnostic test. This will allow the user to plot both the true modelled prevalence, and adjusted observed/apparent prevalence, as described in Figure 3 below
+
+```{r echo=FALSE, out.width='100%'}
+knitr::include_graphics('./diagnostic_adjustment.jpg')
+```
+
+To run this in the model, this can be specified as (e.g. with a hypothetical 90% sensitivity and 80% specificty of a porcine cysticercosis diagnostic test) :
+
+```{r}
+# Set up the model, specifying diagnostic parameters for pig cysticerosis prevalence adjustment
+ s1 <- set_up(PC_sens=0.9, PC_spec=0.8)
+# Run the model, with new input parameters and initial conditions
+ m4 <- run_model(time = 50, burn_in = 100, params = s1[[1]], initial_states = s1[[2]])
+```
+
+## Specifying the non age-structured model
+
+The age-structured model can be configured to remove age-structuring, and therefore provide the original model presented in Winskill et al. 2017. To specify the non age-structured model, the following parameter inputs need to be included in `set_up()`. :
+
+```{r}
+# Set up the model, specifying the non age-structured model
+ s1 <- set_up( number_age_classes_human = 1, number_age_classes_pig = 1, slaughter_age_min = 0)
+# Run the model, with new input parameters and initial conditions
+ m5 <- run_model(time = 50, burn_in = 100, params = s1[[1]], initial_states = s1[[2]])
+```
+This will collapse age groups so that only one age group is modelled for human and pig state variables:
+
+```{r, results='asis', echo=FALSE}
+knitr::kable(utils::head(m5[, c(1,5:14)], 10), caption = 'All state variables, collapsed to one age group', digits = 1)
+```
+
+If the user wishes to specify the complete original model presented in Winskill et al. 2017 with no age-strcuturing, and no pre-patent compartment in pigs (Pp compartments in Figure 1), then the user can set the rate at whicb pigs progress from the pre-patent compartment to a very high value (by setting the average duration pig remain in the pre-patent compartment parameter `psi`to a very small value i.e. 1 / 0.00000001) to effectively negate the use of this compartment as follows:
```

Comment:
strcuturing, whicb. Some typos throughout

Reply:
Thanks, will check and amend typos.